resources/views/ws/ghap/places/show.blade.php

```diff
@@ -42,9 +47,9 @@
                         WS Feed
                     </button>
                     <div class="dropdown-menu" aria-labelledby="wsfeedDropdown">
-                        <a class="dropdown-item grab-hover" href="{{url()->full()}}&format=kml">KML</a>
-                        <a class="dropdown-item grab-hover" href="{{url()->full()}}&format=csv">CSV</a>
-                        <a class="dropdown-item grab-hover" href="{{url()->full()}}&format=json">GeoJSON</a>
+                        <a class="dropdown-item grab-hover" href="{{ $url . $separator }}format=kml">KML</a>
+                        <a class="dropdown-item grab-hover" href="{{ $url . $separator }}format=csv">CSV</a>
+                        <a class="dropdown-item grab-hover" href="{{ $url . $separator }}format=json">GeoJSON</a>

```

Comment:
Ok, now I see these links should work with multiple places result page as well. Please revert the change back as the redirection should handle that.

Reply:
if we click the 'WS FEED' button at URL' places/{uid}' . the URL will be wrong: 'places/{uid}&format=json' as there are no parameters